### PR TITLE
Add the missing functonality to add an attachment or inline image to …

### PR DESCRIPTION
**Pull request by ukmadlz**
…the FormData as reported in #30

### CODE REVIEW

**Review by strosek — APPROVED** [review]
Looks good to me. Thanks!

File: src/apis/email.ts [hunk 1]
```diff
@@ -59,6 +59,34 @@ class Email {
           'Email must contain at least one of these (text, html or templateId).'
         );
 
+      if (email.attachment) {
+        Validator.array(email.attachment, 'email.attachment');
+        email.attachment.forEach((attachmentObject: any) => {
+          Validator.requiredString(
```
Comment (pedrosimoessss) [inline src/apis/email.ts:65]: Shouldn't the attachment be a `Buffer` / `Uint8Array`?

File: test/apis/email.test.ts [hunk 2]
```diff
@@ -84,6 +85,64 @@ describe('Email', () => {
...
+      password: PASSWORD,
+    });
+    await email.send({
+      to: 'test@example.com',
+      from: 'Tests <testing@example.com>',
+      subject: 'Testing',
+      text: 'hello world',
+      attachment: [
+        {
+          data: Fs.readFileSync('./test/apis/email.test.ts'),
```
Comment (pedrosimoessss) [inline test/apis/email.test.ts:104]: 🤔 very strange that this test is green, considering that the following test fails:
```
  it('throws if input is invalid', () => {
      expect(() => Validator.requiredString(Fs.readFileSync('./test/apis/email.test.ts'))).toThrowError();
  });
```
maybe some weird jest concurrent thing? 😅

File: test/apis/email.test.ts [hunk 3]
```diff
@@ -51,7 +171,395 @@ describe('Email', () => {
...
+    });
+
+    expect(axios.post).toHaveBeenCalledWith(
+      '/email/3/send',
+      jasmine.any(FormData),
+      jasmine.anything()
+    );
+  });
+
+  it('exposes a send method that accepts an track argument', async () => {
```
Comment (juliacodez) [inline test/apis/email.test.ts:404]: a track agrument

File: test/apis/email.test.ts [hunk 4]
```diff
@@ -51,7 +171,395 @@ describe('Email', () => {
...
+    });
+
+    expect(axios.post).toHaveBeenCalledWith(
+      '/email/3/send',
+      jasmine.any(FormData),
+      jasmine.anything()
+    );
+  });
+
+  it('exposes a send method that accepts an callbackData argument', async () => {
```
Comment (juliacodez) [inline test/apis/email.test.ts:380]: a callbackData

File: test/apis/email.test.ts [hunk 5]
```diff
@@ -51,7 +171,395 @@ describe('Email', () => {
...
+    });
+
+    expect(axios.post).toHaveBeenCalledWith(
+      '/email/3/send',
+      jasmine.any(FormData),
+      jasmine.anything()
+    );
+  });
+
+  it('exposes a send method that accepts an notifyContentType argument', async () => {
```
Comment (juliacodez) [inline test/apis/email.test.ts:356]: a notifyContentType. Check throughout

**Review by juliacodez — CHANGES_REQUESTED** [review]
Double-check all text for correct use of articles. Looks good otherwise.

**Review by juliacodez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**